### PR TITLE
Release 2.2.0

### PR DESCRIPTION
**Pull request by cerebrl**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
 # Changelog
 
-## Unreleased
+## [2.2.0] - 2020-12-18
 
 ### Added
 
 - Added email suspended node support in sample apps
+- Added support for `TextOutputCallback` `messageType 4` behind a configuration option that is disabled by default
+- Support for TypeScript 4.0
```
Comment (spetrov) [inline CHANGELOG.md:10]: Do you think it worth mentioning the improvements related to the combinations of nodes within a page node? (SDKS-811)

**Review by spetrov — CHANGES_REQUESTED** [review]
Can you also update the supported browsers in the readme file? We tested with: 

Chrome 87
Firefox 84
Safari 14
Edge (Chromium) 87
Legacy Edge 44 (with polyfills)
(IE11 statement is up to date)

File: CHANGELOG.md (hunk 1 above)
Comment (cerebrl) [reply CHANGELOG.md:10]: It was already mentioned under the Fixed header: https://github.com/ForgeRock/forgerock-javascript-sdk-ui/pull/40/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR13

**Comment by cerebrl** [comment]
@spetrov Updated the README.md and linked to the mention of the page node callback fix.

File: CHANGELOG.md (hunk 1 above)
Comment (spetrov) [reply CHANGELOG.md:10]: I must have been blind... not sure how I missed it...

**Review by spetrov — APPROVED** [review]
Looks good to me! 👍

**Pull request merged**